source/val/validate_annotation.cpp

```diff
@@ -200,7 +200,8 @@ spv_result_t ValidateDecorationTarget(ValidationState_t& _, SpvDecoration dec,
             sc != SpvStorageClassHitAttributeKHR &&
             sc != SpvStorageClassCallableDataKHR &&
             sc != SpvStorageClassIncomingCallableDataKHR &&
-            sc != SpvStorageClassShaderRecordBufferKHR) {
+            sc != SpvStorageClassShaderRecordBufferKHR &&
+            sc != SpvStorageClassHitObjectAttributeNV) {
           return _.diag(SPV_ERROR_INVALID_ID, target)
                  << _.VkErrorID(6672) << _.SpvDecorationString(dec)
```

Comment:
There should be a spec update for ` VUID-StandaloneSpirv-Location-06672` to add `HitObjectAttributeNV` storage class

Reply:
See https://gitlab.khronos.org/vulkan/vulkan/-/merge_requests/5538